### ISSUE TITLE
NPE when opening a dialog of "Add fields changing sequence" step

### ISSUE THREAD

**Issue by HiromuHota** [HiromuHota/pentaho-kettle#42]
The following error can be seen in the log when opening a dialog of "Add fields changing sequence" step
```
Exception in thread "Thread-25" java.lang.NullPointerException
	at org.pentaho.di.ui.trans.steps.fieldschangesequence.FieldsChangeSequenceDialog$7.run(FieldsChangeSequenceDialog.java:298)
	at java.lang.Thread.run(Thread.java:745)
```

**Comment by HiromuHota** [comment]
This also happens for "String operations"
```
Exception in thread "Thread-72" java.lang.NullPointerException
	at org.pentaho.di.ui.trans.steps.stringoperations.StringOperationsDialog$7.run(StringOperationsDialog.java:293)
	at java.lang.Thread.run(Thread.java:745)
```

**Comment by HiromuHota** [comment]
Resolved by 425d46356bd51074d8cbd524b6b925fbff4c79e8.
